models/migrations/v47.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"github.com/go-xorm/xorm"
+)
+
+func addLoginSourceLdapPublicSSHKeySyncEnabled(x *xorm.Engine) error {
+	// LoginSource see models/login_source.go
+	type LoginSource struct {
+		IsLdapPublicSSHKeySyncEnabled bool `xorm:"INDEX NOT NULL DEFAULT false"`
```

Comment:
I don't see this field used any more, so this sync should be just replaced with added LoginSourceID column for PublicKey

Reply:
Oh yeah, right. I changed this behaviour, now the sync is enabled as soon as it's configured and there's no longer any option to enable/disable the sync from the app.ini file. I've just somehow forgotten that I did this, and to remove this part of the code as well. Grr :) Nice catch, thnx!